src/app/shared/entry.ts

```diff
@@ -196,4 +197,55 @@ export abstract class Entry implements OnInit, OnDestroy, AfterViewInit {
     }
     return selectedVersion;
   }
+
+  // Selects a tab of index tabIndex
+  selectTab(tabIndex: number) {
+    this.entryTabs.tabs[tabIndex].active = true;
+  }
+
+  // Sorts two entries by last modified, and then verified
+  entryVersionSorting(a: any, b: any): number {
+    if (a.last_modified > b.last_modified) {
+      return -1;
+    } else if (a.last_modified < b.last_modified) {
+      return 1;
+    } else {
+      if (a.verified && !b.verified) {
```

Comment:
Oh, I see there might be ambiguity in language here. 
I meant algorithmically, one should sort the table first by last modified and then by verified. 
I think this algorithm sorts once by modified date first and then if those are tied, sort by verified which isn't quite right. But it is worth double-checking

Reply:
Oops ya I assumed the second case, I'll fix it.